### PR TITLE
Redirect BiomeLayers.getModdedBiomeSize to PatchworkBiomeLayers

### PR DESCRIPTION
**Pull request by leo60228**

### CODE REVIEW

File: src/main/java/com/patchworkmc/patch/LevelGeneratorTypeTransformer.java [hunk 1]
```diff
@@ -27,15 +27,6 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
 			super(Opcodes.ASM7, methodVisitor);
 		}
 
-		@Override
-		public void visitTypeInsn(int opcode, String type) {
```
Comment (coderbot16) [inline src/main/java/com/patchworkmc/patch/LevelGeneratorTypeTransformer.java]: Why is this removed? As far as I can tell, it's needed for level generator types to properly work.

File: src/main/java/com/patchworkmc/patch/BiomeLayersTransformer.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		return new MethodTransformer(super.visitMethod(access, name, descriptor, signature, exceptions));
+	}
+
+	private static class MethodTransformer extends MethodVisitor {
+		MethodTransformer(MethodVisitor methodVisitor) {
+			super(Opcodes.ASM7, methodVisitor);
+		}
+
+		@Override
+		public void visitTypeInsn(int opcode, String type) {
```
Comment (coderbot16) [inline src/main/java/com/patchworkmc/patch/BiomeLayersTransformer.java]: I don't think that anyone will be needing to initialize BiomeLayers? Or at least, it won't need to be redirected.

**Review by coderbot16 — CHANGES_REQUESTED** [review]
Looks good to me, but it looks like you removed the wrong NEW redirect.

**Comment by leo60228** [comment]
Oops... I'll fix that after I eat breakfast.

**Comment by coderbot16** [comment]
Sounds good!

File: src/main/java/com/patchworkmc/patch/BiomeLayersTransformer.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	private static final String BIOME_LAYERS_NAME = "net/minecraft/class_3645";
+	private static final String PATCHWORK_BIOME_LAYERS_NAME = "net/patchworkmc/api/levelgenerators/PatchworkBiomeLayers";
+
+	public BiomeLayersTransformer(ClassVisitor classVisitor) {
+		super(Opcodes.ASM7, classVisitor);
+	}
+
+	@Override
+	public void visit(int version, int access, String name, String signature, String superName, String[] interfaces) {
+		super.visit(version, access, name, signature, BIOME_LAYERS_NAME.equals(superName) ? PATCHWORK_BIOME_LAYERS_NAME : superName, interfaces);
```
Comment (coderbot16) [inline src/main/java/com/patchworkmc/patch/BiomeLayersTransformer.java]: This probably isn't needed either

**Review by coderbot16 — CHANGES_REQUESTED** [review]
Looks mostly good, just noticed one other small issue.

**Review by coderbot16 — APPROVED** [review]
Looks good to me!

**Pull request merged**